.github/workflows/browser_test.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow will do a clean install, start the selenium server, and run
+# all of our browser based test
+
+name: Run browser test nightl
+
+on:
+  # Trigger the workflow nightly
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  build:
+    timeout-minutes: 10
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        # TODO (#2114): re-enable osx build.
+        # os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest]
+        node-version: [16.x, 18.x, 20.x]
+        # See supported Node.js release schedule at
+        # https://nodejs.org/en/about/releases/
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Reconfigure git to use HTTP authentication
+        run: >
+          git config --global url."https://github.com/".insteadOf
+          ssh://git@github.com/
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Npm Install
+        run: npm install
+
+      - name: Linux Test Setup
+        if: runner.os == 'Linux'
+        run: source ./tests/scripts/setup_linux_env.sh
+
+      - name: MacOS Test Setup
+        if: runner.os == 'macOS'
+        run: source ./tests/scripts/setup_osx_env.sh
+
+      - name: Run
+        run: npx mocha ./tests/browser/test --config ./tests/browser/test/.mocharc.js
```

Comment:
Can you add this command as a [package.json script](https://github.com/google/blockly/blob/9442b73d99fb63da4526c651ac41762e458b4ed7/package.json#L19) and run that instead? Good options might be `tests:selenium`, `tests:integration`, or anything of the form `tests:*`.